### PR TITLE
Container requirement updates jcayouette

### PR DESCRIPTION
**Pull request by jcayouette**
# Containerization Improvements

- [x] Adjust repository from master to stable
- [x] Storage / Permissions volume cleanup
- [x] Requirements pages cleanup  
- [x] Deprecate additional unified installer content
- [x] Adjust ports for containerization
- [x] Update changelog

## Branches
- master

## References:
- https://github.com/uyuni-project/uyuni-tools/blob/main/shared/utils/ports.go
- installation-and-upgrade/ports.adoc

### CODE REVIEW

File: modules/installation-and-upgrade/pages/general-requirements.adoc [hunk 1]
```diff
@@ -4,7 +4,6 @@
 Before you begin installation, ensure that you have:
 
 * Current {scc} organization credentials
-* Access to installation media
```
Comment (cbosdo) [inline modules/installation-and-upgrade/pages/general-requirements.adoc:7]: Why do you remove this one? You still need access to the media of the Server extension be it via SCC or using an ISO image.

File: modules/installation-and-upgrade/pages/hardware-requirements.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ For {productname} {smr} hardware requirements, see xref:retail:retail-requiremen
 [[server-hardware-requirements]]
 == Server Hardware Requirements
 
-{productname} Server stores packages in the [path]``/var/spacewalk/`` directory.
+By default the {productname} Server container stores packages in the [path]``/var/spacewalk/`` directory.
```
Comment (cbosdo) [inline modules/installation-and-upgrade/pages/hardware-requirements.adoc:23]: This is the path inside the container, you probably want to highlight the path on the host and the `--podman-mount-[cache|postgresql|spacewalk]` options to override it.

File: modules/installation-and-upgrade/pages/general-requirements.adoc (hunk 1 above)
Comment (jcayouette) [reply modules/installation-and-upgrade/pages/general-requirements.adoc:7]: This list item was referencing the unified installer and installation medium.

In the deployment guides we provide the task for connecting to SCC during installation on SLE micro 5.5 or Leap Micro 5.5, the input of reg keys etc.

The general requirements guide is not the place for this. (Or am I mistaken?)

File: modules/installation-and-upgrade/pages/general-requirements.adoc [hunk 3]
```diff
@@ -13,9 +12,12 @@ This section contains more information on each of these requirements.
 For a complete list of supported clients and features, see xref:client-configuration:supported-features.adoc[].
 
 
+.Base operating system for {productname} {productnumber} containerized Server
 [IMPORTANT]
 ====
-{productname} {productnumber} is based on SLES {sles-version} {sp-base} as the host operating system.
+For {productname} {productnumber}, the base operating system of the container is {sles} {bci-suma}.
+You can find these keys in your {scc} account.
+
```
[the remark this answers is not in the document]
Comment (jcayouette) [reply modules/installation-and-upgrade/pages/general-requirements.adoc:20]: There is an ifeval conditional for both products. This clarifies distinctions between the two.

File: modules/installation-and-upgrade/pages/hardware-requirements.adoc [hunk 4]
```diff
@@ -165,39 +165,41 @@ For more information about managing a large number of clients, see xref:speciali
 | Minimum 40{nbsp}GB
 
 |
-| [path]``/srv``
+| [path]``/var/lib/containers/storage/volumes/srv-www``
```
Comment (cbosdo) [inline modules/installation-and-upgrade/pages/hardware-requirements.adoc:168]: Do you really want to add this one here? `/srv` could contain some data in the past, but not the `srv-www` volume contains the bootstrap repos, not much more.

**Review by cbosdo — COMMENTED** [review]
Shouldn't you mention the `mgradm podman install` options to change the big volumes location?
* `--podman-mount-cache`
* `--podman-mount-postgresql`
* `--podman-mount-spacewalk`

File: modules/installation-and-upgrade/pages/hardware-requirements.adoc (hunk 2 above)
Comment (jcayouette) [reply modules/installation-and-upgrade/pages/hardware-requirements.adoc]: Added your comment to my notes for storage docs.

File: modules/installation-and-upgrade/pages/hardware-requirements.adoc (hunk 4 above)
Comment (jcayouette) [reply modules/installation-and-upgrade/pages/hardware-requirements.adoc:168]: Left as is after our discussion. Boostrap content, distributions, and containers are stored here so it should be considered.

**Comment by jcayouette** [comment]
> Shouldn't you mention the `mgradm podman install` options to change the big volumes location?
> 
> * `--podman-mount-cache`
> * `--podman-mount-postgresql`
> * `--podman-mount-spacewalk`

Yes Added this to my storage doc notes. Its next on the list.

**Pull request merged**